qutebrowser/browser/downloads.py

```diff
@@ -46,11 +46,6 @@ class ModelRole(enum.IntEnum):
 
     item = Qt.UserRole
 
-
-# Remember the last used directory
-last_used_directory = None
```

Comment:
I'm not sure if this is standard practice, but I would prefer to leave this initialization here, if only to show that such a variable exists in this file (instead of inferring it from the globals). 

It can also cause weird issues if, for example, `init` isn't run before other methods.

Reply:
I added this initialization back in